### PR TITLE
Some simple pyplot doc improvements

### PR DESCRIPTION
**Pull request by timhoffm**

### CODE REVIEW

File: lib/matplotlib/pyplot.py [hunk 1]
```diff
@@ -329,6 +330,9 @@ def gci():
     """
     Get the current colorable artist.
 
+    Historically, the only colorable artists were images; hence the name
+    ``gci`` (get current image).
```
Comment (anntzer) [inline lib/matplotlib/pyplot.py]: perhaps move this to the Notes section

File: lib/matplotlib/pyplot.py [hunk 2]
```diff
@@ -602,7 +610,19 @@ def get_figlabels():
 
 
 def get_current_fig_manager():
-    """Return ``gcf().canvas.manager``, the current figure's manager."""
+    """
+    Return the figure manager of the current figure.
+
+    The figure manager can be regarded as a container for the actual
```
Comment (anntzer) [inline lib/matplotlib/pyplot.py]: "can be regarded as" -> "is"

File: lib/matplotlib/pyplot.py (hunk 1 above)
Comment (timhoffm) [reply lib/matplotlib/pyplot.py]: Sure.

File: lib/matplotlib/pyplot.py (hunk 2 above)
Comment (timhoffm) [reply lib/matplotlib/pyplot.py]: Done.

**Pull request merged**